tools/php_api_ref/phpdoc.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+set +x;
+
+AUTH_JSON=${1:-~/.composer/auth.json}; # Path to an auth.json file allowing to install the targeted edition and version
+OUTPUT_DIR=${2:-./docs/api/php_api/php_api_reference}; # Path to the directory where the built PHP API Reference is hosted
+
+DXP_EDITION='commerce'; # Edition from and for which the Reference is built
+DXP_VERSION='4.6.*@dev'; # Version from and for which the Reference is built
+DXP_EDITIONS=(oss headless experience commerce); # Available editions ordered by ascending capabilities
+TMP_DXP_DIR=/tmp/ibexa-dxp-phpdoc; # Absolute path of the temporary directory in which Ibexa DXP will be installed and the PHP API Reference built
+PHPDOC_VERSION='3.4.3'; # Version of phpDocumentor used to build the Reference
```

Comment:
Found a bug in this version ([issue 3633](https://github.com/phpDocumentor/phpDocumentor/issues/3633)) that should be fixed in the next one. Wait for next [release](https://github.com/phpDocumentor/phpDocumentor/releases).

```suggestion
PHPDOC_VERSION='3.4.4'; # Version of phpDocumentor used to build the Reference
```